src/libasr/runtime/lfortran_intrinsics.c

```diff
@@ -737,6 +737,19 @@ LFORTRAN_API int32_t _lpython_bit_length8(int64_t num)
     return res;
 }
 
+LFORTRAN_API char* _lpython_str_capitalize(char* str) {
+    int len = strlen(str);
+    char *res = (char*)malloc(len+1);
+    strcpy(res, str);
+    if (len != 0) {
+        if(res[0] >= 'a' && res[0] <= 'z') {
+            res[0] -= 32;
+        }
+    }
+    res[len] = '\0';
+    return res;
+}
```

Comment:
This function I think we can implement in LPython, so we should, and remove it from ASR.

We should only put things into ASR that cannot be implemented in LPython.

Reply:
i've implemented  like `_lpython_floordiv` is that the correct way?